SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/CrossServerPossessionTest.cpp

```diff
@@ -53,16 +52,19 @@ void ACrossServerPossessionTest::PrepareTest()
 				{
 					AssertTrue(PlayerController->HasAuthority(), TEXT("PlayerController should HasAuthority"), PlayerController);
 					AssertFalse(Pawn->HasAuthority(), TEXT("Pawn shouldn't HasAuthority"), Pawn);
-					PlayerController->UnPossess();
-					PlayerController->RemotePossessOnServer(Pawn);
+					if (!(Pawn->HasAuthority()))
+					{
+						PlayerController->UnPossess();
+						PlayerController->RemotePossessOnServer(Pawn);
+					}
 				}
 			}
 		}
 		FinishStep();
 	});
 
 	AddStep(
-		TEXT("Check test result"), FWorkerDefinition::Server(1),
+		TEXT("Check test result"), FWorkerDefinition::Server(4),
```

Comment:
Why this server change?

Reply:
The initial test was passing because the assert was never reached because the condition in the "if" was never true, as the PlayerController migrates to Server 4 (that's where the pawn is spawned) so it would be nullptr when accessed from Server 1.